data/blog/webview2.md

```diff
@@ -0,0 +1,112 @@
+- electron
+date: '2021-07-08'
+---
+
+Over the past week, we’ve received several questions about the differences between the new [WebView2](https://docs.microsoft.com/en-us/microsoft-edge/webview2/) and Electron.
+We want to provide some early information on the similarities and differences of the two solutions.
+
+---
+
+## Architecture Overview
+
+Electron and WebView2 both build from the Chromium source for rendering web content.
+Strictly speaking, WebView2 builds from the Edge source, but Edge is built using a fork of the Chromium source.
+Electron does not share any DLLs with Chrome. 
+WebView2 binaries hard link against Edge (Stable channel as of Edge 90), so they share disk and some working set.
+
+Electron apps always bundle and distribute the exact version of Electron with which they were developed.
+WebView2 has two options in distribution.
+You can bundle the exact WebView2 library your application was developed with, or you can use a shared-runtime version that may already be present on the system.
+WebView2 provides tools for each approach, including a bootstrapping installer in case the shared runtime is missing.
+WebView2 is shipped _inbox_ starting with windows 11.
+
+Applications that bundle their frameworks are responsible for updating those frameworks, including minor security releases.
+For apps using the shared WebView2 runtime, WebView2 has its own updater, similar to Chrome or Edge, that runs independent of your application.
+Your app will transparently inherit any updates every time it launches.
+Neither Electron nor WebView2 is managed by Windows Update.
+
+Both Electron and WebView2 inherit Chromium’s multi-process architecture - namely, a single main process that communicates with one-or-more renderer processes.
+These processes are entirely separate from other applications running on the system.
+Every Electron application is a separate process tree, containing a root browser-process, some utility processes, and zero or more render processes.
+WebView2 apps use the same [user data folder](https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/user-data-folder) (like a suite of apps would do), share non-renderer processes.
+WebView2 apps using different data folders do not share processes.
+
+* ElectronJS Process Model:
+
+    ![ElectronJS Process Model Diagram](/images/Electron-Architecture.png)
+* WebView2 Based Application Process Model:
+
+    ![WebView2 Process Model Diagram](/images/WebView2-Architecture.png)
+
+Read more about [WebView2’s process model](https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/process-model#:~:text=WebView2%20uses%20the%20same%20process%20model%20as%20the,other%20utility%20processes%20as%20described%20in%20that%20article.) and [Electron’s process model](https://www.electronjs.org/docs/tutorial/process-model#:~:text=Process%20Model%20Electron%20inherits%20its%20multi-process%20architecture%20from,applied%20in%20the%20minimal%20quick%20start%20app%20.) here.
```

Comment:
```suggestion
Read more about [WebView2’s process model](https://docs.microsoft.com/en-us/microsoft-edge/webview2/concepts/process-model) and [Electron’s process model](https://www.electronjs.org/docs/tutorial/process-model) here.
```

The current links will highlight parts of the page. Don't know if that's an Edge only feature or works also with Chrome and if we want that (I personally prefer cleaner links).